src/main.cpp

```diff
@@ -24,37 +60,30 @@ int main(int argc, char **argv)
 	std::srand(std::time(nullptr));
 
 	OpenglBoilerplate boilerplate = {};
-	Map map = {};
-	Enemy enemy = {vec3{0,0,-50}};  // TODO: get rid of hard-coded place
-	Camera camera = {};
-	PhysicsObject player = {vec3{0,map.get_height(0, 0),0}};
+	GameState game_state = {};
 
 	int zdir = 0;
 	int angledir = 0;
-	float angle = 0;
 
-	double last_time = counter_in_seconds();
+	double last_time = counter_in_seconds() - 1e-5;  // Make sure at least one physics iteration runs below
```

Comment:
This change was intended to fix a bug that didn't get fixed. Sometimes when you start the game, you see weird stuff.

```suggestion
	double last_time = counter_in_seconds();
```